cfme/automate/dialog_element.py

```diff
@@ -54,6 +55,7 @@ def is_displayed(self):
 class EditElementView(ElementForm):
     save_button = Button('Save')
     reset_button = Button('Reset')
+    dd = DragandDrop()
```

Comment:
**REQUIRED:** dd is not descriptive enough I'm afraid

Reply:
Changed